### ISSUE TITLE
Issues with MathJax preview

### ISSUE THREAD

**Issue by astrozot** [James-Yu/LaTeX-Workshop#1411]
**Disable all the other extensions except for LaTeX Workshop, and check that you still see this issue.**

Done.

You still see this issue?: Yes

**Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.**

Done

**Describe the bug [Required]**
I have one issue with the MathJax Preview. I see an incomplete or strange display of equations depending on the position of the cursor. 

**To Reproduce**

Below I indicate the position of the cursor with a vertical bar in simple LaTeX document. 

1. Within inline equations, the MathJax preview is behaving differently depending on the position of the cursor. In particular, if the cursor is just before the first dollar sign, the first character after the dollar sign is missing in the preview: |$abc$ -> bc

2. If the cursor is just after the dollar sign, the equation is correct $|abc$ -> abc.

3. If the cursor is in the middle of the equation, the \ddagger symbol appears at the location of the cursor and the first character is missing: $ab|c$ -> b \ddagger c

4. In all other cases, the first character is missing.

Exactly the same happens for equations introduced by the \[ ... \] symbol.

**Expected behavior**
I would not expect to see a \ddagger character, nor to have missing characters in equations.

**Logs [Required]**

<details>
<summary>LaTeX Workshop Output [Required]</summary>
[19:41:51] Initializing LaTeX Workshop.
[19:41:51] Creating LaTeX Workshop http and websocket server.
[19:41:51] LaTeX Workshop initialized.
[19:41:51] Found root file from active editor: /Users/mlombard/TeX/art65/art.tex
[19:41:51] Root file changed from: undefined. Find all dependencies.
[19:41:51] Instantiating a new file watcher for /Users/mlombard/TeX/art65/art.tex
[19:41:51] Parsing /Users/mlombard/TeX/art65/art.tex
[19:41:51] Cannot find .bib file dark-refs
[19:41:51] Parsing /Users/mlombard/TeX/art65/art.fls to compute dependencies
[19:41:51] Adding /Users/mlombard/TeX/inputs/aa.cls to file watcher.
[19:41:51] Cannot find .bib file dark-refs
[19:41:51] Server created on 127.0.0.1:54988
[19:41:51] Snippet data loaded.
[19:41:51] LaTeX Workshop version: 7.0.0
</details>

<details>
<summary>Developer Tools Console [Required]</summary>
Really do not see anything suspicious; anyway, I cannot past things related to the display because as soon as I move the mouse to the developer console the preview disappear, as expected.
</details>

**Screenshots**
<img width="1680" alt="Screen Shot 2019-05-27 at 20 25 57" src="https://user-images.githubusercontent.com/2106594/58434788-af50e780-80bd-11e9-8a6a-2d66aa536974.png">

**Desktop [Required]:**
 - OS: Mac OS 10.14.15
 - VS Code version: 1.34.0
 - Extension version: 7.0.0

**Additional context**
I should add that the preview does not seem to work properly if an inline equation spans multiple lines.

**Comment by trevorgunn** [comment]
The `\ddagger` behaviour is intended. You can turn it off with the [latex-workshop.hover.preview.cursor.enabled](https://github.com/James-Yu/LaTeX-Workshop/wiki/Hover#latex-workshophoverpreviewcursorenabled) setting.

I'm not sure what is happening with the missing characters. I can't replicate the behaviour on Windows or Linux. Can you replicate this with an empty document? Or maybe you have something in your settings?

> I should add that the preview does not seem to work properly if an inline equation spans multiple lines.

Also intended. Multiline inline environments are hard to parse. E.g.
```latex
$
a
$
b
$
c
$
d
$
e
$
f
```
I'm sure it could be made to work, but I don't think anyone is particularly interested in working on it.
